typescript/agentkit/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # AgentKit Changelog
 
+## 0.5.1
```

Comment:
Please do not update CHANGELOG directly, as this is generated during releases.

[Generate a changelog entry via running `npx @changesets/cli` from the `/typescript` folder](https://github.com/coinbase/agentkit/blob/main/CONTRIBUTING-TYPESCRIPT.md#changelog)

Reply:
yes, applied change